pages/code-review.md

```diff
@@ -24,10 +24,34 @@ goals when reviewing code are to:
 - cultivate the ability to critique one’s code more strongly;
 - encourage open communication between the entire team.
 
+## Discuss code review with your team
+
+Every individual and team has a different set of expectations around code
+review. Discussing these expectations explicitly is the best way to make sure
+that everyone is on the same page.
+
+Bonus points for documenting and/or automating the outcomes of your discussion.
+
+Some guiding questions for your team's code review discussion:
+
+- What does the current code review process, at a high level, look like? Start
+  with the feature request and discuss each step until a feature is considered
+  "done"
```

Comment:
This is a great question!  The definition of "done" is an important distinction in agile, so if a team doesn't have a solid definition of knowing when a story/feature/etc. is truly finished this will hopefully help drive that conversation as well, or at least shed light on the need to do so.